src/main/java/org/ohdsi/cohortcharacterization/CCQueryBuilder.java

```diff
@@ -418,8 +429,8 @@ private String getStrataCohortTable(CohortCharacterizationStrata strata) {
 
 		List<String> queriesToRun = new ArrayList<>();
 		queriesToRun.addAll(getQueriesForPresetAnalyses(jsonObject,cohortDefinitionId, strata));
-		queriesToRun.addAll(getQueriesForCustomDistributionAnalyses(cohortDefinitionId));
-		queriesToRun.addAll(getQueriesForCustomPrevalenceAnalyses(cohortDefinitionId));
+		queriesToRun.addAll(getQueriesForCustomDistributionAnalyses(cohortDefinitionId, strata));
+		queriesToRun.addAll(getQueriesForCustomPrevalenceAnalyses(cohortDefinitionId, strata));
```

Comment:
strate argument can be null, due to this, we have to always do null pointer check, I would suggest using a dummy empty class something like this 
```
	public static  CohortCharacterizationStrata EMPTY_STRATA= new CohortCharacterizationStrata() {
		public String getName() {
			return StringUtils.EMPTY;
		}
		@Override
		public Long getId() {
			return 0L;
		}
	}
```